components/layouts/QiskitOrgMenu.vue

```diff
@@ -1,353 +1,143 @@
-              :class="{
-                'vertical-navigation__item': true,
-                'vertical-navigation__item--active': isCommunityActive()
-              }"
-              to="/education"
-            >
-              Community
-            </nuxt-link>
-            <div
-              v-if="isCommunityActive()"
-              class="vertical-community-navigation"
-            >
-              <nuxt-link
-                v-for="communitySubLink in communitySubLinks"
-                :key="communitySubLink.label"
-                :class="{
-                  'vertical-community-navigation__item': true,
-                  'nuxt-link-active': isActive(communitySubLink.url)
-                }"
-                :to="communitySubLink.url"
-              >
-                {{ communitySubLink.label }}
-              </nuxt-link>
-            </div>
-            <a
-              v-for="link in learnMore"
-              :key="link.label"
-              :class="{
-                'vertical-navigation__item': true,
-                'nuxt-link-active': isActive(link.url)
-              }"
-              :href="link.url"
-              target="_blank"
-              @click="link.segment && $trackClickEvent(link.segment)"
-            >
-              {{ link.label }}
-            </a>
-          </nav>
-        </section>
-        <nuxt-link class="link-to-home" to="/">
-          Qiskit
-        </nuxt-link>
-        <nav class="navigation-group navigation-group--with-separator">
-          <nuxt-link
-            v-for="link in qiskitElements"
-            :key="link.url"
-            :class="{
-              'navigation-group__item': true,
-              'nuxt-link-active': isActive(link.url)
-            }"
-            :to="link.url"
-          >
-            {{ link.label }}
-          </nuxt-link>
-        </nav>
-        <nav class="navigation-group navigation-group--fixed navigation-group--right-aligned">
-          <nuxt-link
-            :class="{
-              'navigation-group__item': true,
-              'navigation-group__item--active': isCommunityActive()
-            }"
-            to="/education"
-          >
-            Community
-          </nuxt-link>
-          <a
-            v-for="link in learnMore"
-            :key="link.label"
-            :class="{
-              'navigation-group__item': true,
-              'nuxt-link-active': isActive(link.url)
-            }"
-            :href="link.url"
-            target="_blank"
-            @click="link.segment && $trackClickEvent(link.segment)"
-          >
-            {{ link.label }}
-          </a>
-        </nav>
-      </div>
-    </div>
-    <div
+  <div class="menu">
+    <section class="menu__mobile" tabindex="-1">
+      <Menu20 class="menu__link" />
+      <MenuLink v-bind="homeLink" />
+      <div class="menu__overlay" />
+      <SidebarMenu />
+    </section>
+    <section class="menu__main-level">
+      <nav class="menu__main-level_on-left">
```

Comment:
According to BEM, [a modifier cannot be used alone](https://en.bem.info/methodology/quick-start/#a-modifier-cant-be-used-alone) and it needs to [semantically describe what the CSS is doing](https://en.bem.info/methodology/faq/#how-to-choose-a-name-for-a-modifier). I think here we only need to name the `<nav>` elements with something like `menu__navigation-section` (perhaps coming with a better name but I hope you get the idea).